packages/common/core-interfaces/src/events.ts

```diff
@@ -0,0 +1,430 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Base interface for event emitters.
+ */
+export interface IEvent {
+	/**
+	 * Base event emitter signature.
+	 *
+	 * @remarks The event emitter polyfill and the node event emitter have different event types:
+	 * `string | symbol` vs. `string | number`.
+	 *
+	 * So for our typing we'll contrain to string, that way we work with both.
+	 *
+	 * @eventProperty
+	 */
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Since this is part of a breaking change anyways, could we replace the instances of `any` in the moved APIs with `unknown`?

Reply:
On that note, if core-interfaces isn't using the strict eslint config yet, that would be a good idea.

Reply:
Good ideas. I don't want to change any to unknown in this change, and I'm reluctant to include it in this work stream at all. This work is strictly focused on copying and deprecating to get to no code sharing. Anything related to making the apis better or different should be considered and funded separately.

Reply:
Same with the lint config. Definite improvement, but different goal and thus out of scope.